.github/workflows/build_nilrt.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       id: submodules
       run: |
         git submodule update --init --recursive
-        echo "::set-output name=grpc-commit::$(git rev-parse --short @:./third_party/grpc)"
```

Comment:
I think @reckenro is more familiar with this code than I am. Are all of these validated by a PR build? What should we want for after submit to make sure everything is bubbling through?

Reply:
This one is validated in the PR build since it uses the grpc-commit as part of the key in the "Cache Host OS gRPC Support" step. And in this build it's still using the grpc-commit (0a82c02) correctly for the key:
```
Run actions/cache@v4
  with:
    path: third_party/grpc/build/
    key: ni/grpc-device-Linux-hostgrpccache-0a82c02
    enableCrossOsArchive: false
    fail-on-cache-miss: false
    lookup-only: false
    save-always: false
```

For the other two, one's used to create the release when a release branch is published and the validate_linux_rt one is run after the release itself is published. I think you could potentially run / trigger those on a fork of the repo but that might be tedious and more effort than it's worth. Otherwise I'd just be prepared to validate at the time of the next release.

Reply:
The changes are small and follow a simple pattern as the change that was tested in the PR. I don't believe it will be much to debug when releasing if there is an issue.